reviewrot/githubstack.py

```diff
@@ -123,16 +129,62 @@ def get_reviews(self, uname, repo_name, state_=None,
                 log.debug("review request '%s' is not %s than specified"
                           " time interval", pr.title, state_)
                 continue
+
+            if last_comment and last_commented:
+                if self.has_new_comments(last_comment.created_at, last_commented):
+                    log.debug("merge request '%s' has new comments  in last %s days", pr.title, last_commented)
+                    continue
+
             res = GithubReview(user=pr.user.login,
                                title=pr.title,
                                url=pr.html_url,
                                time=pr.created_at,
-                               comments=pr.review_comments,
-                               image=pr.user.avatar_url)
+                               comments=pr.review_comments+pr.comments,
+                               image=pr.user.avatar_url,
+                               last_comment=last_comment,
+                               project_name=repo.full_name,
+                               project_url=repo.html_url)
             log.debug(res)
             res_.append(res)
         return res_
 
 
+    def get_last_comment(self, pr):
+
+        review_comments = pr.get_comments()
+        issue_comments = pr.get_issue_comments()
+
+        last_review_comment = None
+        last_issue_comment = None
+
+
+        if review_comments.totalCount > 0:
+            last_review_comment = review_comments.reversed[0]
+
+        if issue_comments.totalCount > 0:
+            last_issue_comment = issue_comments.reversed[0]
+
+        # check which is older if pr has both types of comments
+        if last_issue_comment and last_review_comment:
+            if last_issue_comment.created_at > last_review_comment.created_at:
+                return LastComment(author=last_issue_comment.user.login, body=last_issue_comment.body,
+                                           created_at=last_issue_comment.created_at)
+            else:
+                return LastComment(author=last_review_comment.user.login, body=last_review_comment.body,
+                                       created_at=last_review_comment.created_at)
+
+        # if pr has only one type of comment
+        elif last_issue_comment:
+            return LastComment(author=last_issue_comment.user.login, body=last_issue_comment.body,
+                               created_at=last_issue_comment.created_at)
+
+        elif last_review_comment:
+            return LastComment(author=last_review_comment.user.login, body=last_review_comment.body,
+                               created_at=last_review_comment.created_at)
```

Comment:
The creation of `LastComment` object is duplicated there 4 times. Could you please refactor it?